tests/enforcer/gce_firewall_enforcer_test.py

```diff
@@ -230,9 +231,20 @@ class FirewallRulesTest(ForsetiTestCase):
     def setUp(self):
         """Set up."""
         self.gce_service = mock.MagicMock()
-        self.firewall_api = fe.ComputeFirewallAPI(self.gce_service)
+
+        # Force API_VERSION to always be v1 for these tests, to ensure the generated
```

Comment:
Comment is out of sync. And this is no longer needed since you updated all tests to use 'beta' instead of 'v1' anyway. Please remove the comment and the mock_api_version line.

Reply:
Done